src/@next/Tag/Tag.tsx

```diff
@@ -17,5 +34,41 @@ export const Tag = ({ children, ...props }: TagProps) => {
       children
     );
 
-  return <TagStyle {...props}>{content}</TagStyle>;
+  const removeButton = onRemove ? (
+    <TagRemoveContainerStyle
```

Comment:
can we add `role="button"` to address accessibility risk since we are using div.
then i think we can remove `onMouseDown`, etc and use `:active`, etc in css instead. We will also be able to eliminate the [function](https://github.com/glints-dev/glints-aries/pull/731/files#diff-411820b74eeb16ad04767a6afdd0ed7e90abe1d6ebd043bb0bd85ac575ca5523R36-R45) inside the styled component this way. else, we can address using attributes too but i recommend above if it's possible

Reply:
i've discussed with the designer bernard; we decide to remove the focused state for now

so probably the mouseDown/onMouseUp would be removed

Reply:
the reason why i can't use `:active` for the class selector is because it only applies to the selected element, which in this case, is `TagRemoveContainerStyle`. The problem i'm trying to solve is that, when the user press on the `TagRemoveContainerStyle` element, the parent element's css would change. 

so far in the world of css, i can't find a way to modify parent's element using css purely

Reply:
ahh I see, ok, thanks for giving me more context